src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,182 @@
+package duke;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class Duke {
+    public static ArrayList<Task> tasks = new ArrayList<>();
+    public static int totalTasksDone=0;
+    public static final String LOGO = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+
+    public static final String LINE_BREAK = "\t____________________________________________________________";
+    public static final String YOUR_LIST = "\t_______________________~Your List~__________________________";
+    public enum TaskType{
+        TODO , DEADLINE, EVENT, INVALID
+    }
```

Comment:
Good use of enum!

Reply:
Thanks for the feedback! I'll apply the necessary changes. :))